src/VisualStudio/Core/Def/Implementation/Library/ObjectBrowser/AbstractObjectBrowserLibraryManager.cs

```diff
@@ -101,6 +104,18 @@ private void OnWorkspaceChanged(object sender, WorkspaceChangeEventArgs e)
             }
         }
 
+        private async Task DocumentChangedAsync(Document oldDocument, Document newDocument)
+        {
+            var oldTextVersion = await oldDocument.GetTextVersionAsync(CancellationToken.None).ConfigureAwait(false);
+            var newTextVersion = await newDocument.GetTextVersionAsync(CancellationToken.None).ConfigureAwait(false);
+
+            if (oldTextVersion != newTextVersion)
+            {
+                UpdateClassVersion();
```

Comment:
Right - consider whoever reads these - what happens if they see the state *between* the class version being updated and the member version being updated.  Is that okay, or should both be updated automatically?